qiskit/providers/aer/noise/noise_model.py

```diff
@@ -402,7 +395,7 @@ def reset(self):
         """Reset the noise model."""
         self.__init__()
 
-    def add_basis_gates(self, instructions, warnings=True):
+    def add_basis_gates(self, instructions, warnings=False):
```

Comment:
I personally think `basis_gates` should not be considered in `NoiseModel`.  it should be supplied by a target backend (simulator) at the timing of noise insertion in the future. (I don't touch the issue in this PR)
The following changes just fix a cyclic import issue by removing a warning harmlessly.